package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "floeproject.org",
+  "version": "1.0.0",
+  "description": "FLOE Project Website",
+  "main": "index.js",
+  "scripts": {
+    "build": "eleventy",
+    "eleventy": "eleventy --serve",
+    "eleventyport": "eleventy --serve --port=9778"
+  },
```

Comment:
Should probably add a script for linting. Should add a [`prepare`](https://docs.npmjs.com/misc/scripts) for loading the front end dependencies. That will run on a local `npm install`.

Reply:
Fixed